kubeprod/Makefile

```diff
@@ -1,12 +1,13 @@
-VERSION ?= dev-$(shell date +%FT%T%z)
+VERSION ?= $(shell git describe --tags --dirty)
```

Comment:
Please add this to the top-level makefile for consistency.

Reply:
Oh, I missed the top-level Makefile.  I've added the new definition of VERSION & GIT_TAG, and also changed a few places where VERSION should maybe be GIT_TAG.

I think we still expect VERSION==GIT_TAG for actual releases though.  .. and given the difficulty in testing these bits of the release process, I'm ok reverting those changes (to continue just using VERSION) if you want me to do so.